### PR TITLE
fix bug in send elements default implementation

### PR DESCRIPTION
**Pull request by erohmensing**
**Proposed changes**:
- fixed parenthesis in the wrong place 
- fixes https://rasa-hq.slack.com/archives/C36SS4N8M/p1566817084007700

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -35,6 +37,7 @@ Fixed
 - ``'/restart'`` will now also restart the bot if the tracker is paused
 
 
+
```
Comment (erohmensing) [inline CHANGELOG.rst]:
```suggestion
```

**Comment by erohmensing** [comment]
@akelad ok properly tested now, sorry about that 😅

File: rasa/core/channels/channel.py [hunk 2]
```diff
@@ -244,12 +244,15 @@ def name(cls):
 
         Default implementation will just post the elements as a string."""
 
+        # we can't pass the empty "buttons" key of the message through to send_text_with_buttons()
+        kwargs.pop("buttons", None)
```
Comment (akelad) [inline rasa/core/channels/channel.py]: wait what?

Comment (erohmensing) [reply rasa/core/channels/channel.py]: it has to do with the message that gets sent in through the kwargs, as it's the whole message [here](https://github.com/RasaHQ/rasa/blob/master/rasa/core/channels/channel.py#L184) 

where the message comes from [this](https://github.com/RasaHQ/rasa/blob/3e62036bf9fd458fb5ae81d3b88079604adee357/rasa/core/actions/action.py#L111) being transfromed [here](https://github.com/RasaHQ/rasa/blob/8c18a3409d7609cb9c40be9c5107d443b80c4e24/rasa/core/events/__init__.py#L375) into a dict of all of the message data together. so the kwargs have a `"buttons: None"` arg that would get re-sent through `**kwargs` and conflict with the button argument in [here](https://github.com/RasaHQ/rasa/blob/master/rasa/core/channels/channel.py#L215)

Comment (erohmensing) [reply rasa/core/channels/channel.py]: @akelad thoughts?

Comment (akelad) [reply rasa/core/channels/channel.py]: ahh got it, sorry was occupied with other stuff :D

**Comment by akelad** [comment]
@erohmensing you can merge!

**Pull request merged**